01_fib/leonarj4/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import ("fmt")
+
+func fib(n int) int {
+	if (n == 0) {
+		return n + 1
+	}
+	x,y := n, n-1 
+	return x + y
+}
+
+func main() {
```

Comment:
The lab calls for calling fib(7) in main. The printing is done in the fib(n) function and does not return the final Fibonacci number but the whole sequence. So as fib(1) would just print 1, fib(3) would be
 1
 1
 2